jenkins/scripts/deploy-dev.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env sh
+
+#
+# Will change this so it copies the latest build created by Jenkins
+# so the DEV server won't have to rebuild an already built project.
+#
+
+ORG_PATH="~/JavaPlebes"
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2088
Do we need to take a relook at this?